packages/vscode-extension/src/service/VSTranslationService.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ ** Copyright (c) 2020 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+
+const path = require('path');
+import { TranslationService } from '../util/ExtensionUtil';
+
+const filePath = '../../../vscode-extension/messages.json';
```

Comment:
isn't this `../../messages.json` ?

Reply:
No, the problem is that this is passed to TranslationService.js in the node modue, so that's why we should pass the location of vscode extension as this